ocs_ci/ocs/cluster.py

```diff
@@ -650,6 +762,214 @@ def time_taken_to_complete_rebalance(self, timeout=600):
                 time_taken = time.time() - start_time
                 return (time_taken / 60)
 
+    def osd_health_check(self, count):
+        """
+        OSD health check based on pod count
+        Args:
+            count (int): Expected number of Osd pods
+        Raises:
+            OsdCountException: if osd pod count doesn't match
+        """
+        timeout = 10 * len(self.pods)
```

Comment:
we need to re-visit this method of calculation, as stated here by @mbukatov https://github.com/red-hat-storage/ocs-ci/pull/2287#discussion_r440932218

Reply:
As expecting more time? because we are checking pod status, count after ceph health is OK

Reply:
@shylesh saw you initially added this calculation in https://github.com/red-hat-storage/ocs-ci/pull/362. Could you please elaborate on it?
@prsurve we can continue this discussion in an issue. Will not hold this PR